apollos-church-api/config.yml

```diff
@@ -21,10 +21,6 @@ DATABASE:
     pool:
       max: 39
       min: 1
-    dialectOptions:
```

Comment:
So we need this to be present in production, yet it needs to be removed in dev mode.  I wasn't able to figure out a good way to do that last year.